app/javascript/v3/components/Form/ProfileAvatar.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div
+    class="relative rounded-xl h-[72px] w-[72px] cursor-pointer"
+    @mouseover="onImageOver"
+    @mouseleave="onImageOut"
+  >
+    <img
+      v-if="shouldShowImage"
+      class="rounded-xl h-[72px] w-[72px]"
+      :alt="name"
+      :src="src"
+      draggable="false"
+      @load="onImageLoad"
+      @error="onImageLoadError"
+    />
+    <initials-avatar v-else-if="!shouldShowImage" :name="name" :size="72" />
+
+    <input
+      ref="fileInputRef"
+      type="file"
+      accept="image/png, image/jpeg, image/jpg, image/gif, image/webp"
+      style="display: none"
```

Comment:
@muhsin-k Instead of adding styles for hidding the input, let us add `hidden` attribute in the `<input>` element. It will instruct the browser not to display the element on the page.